docs/tutorial_tripole_grid.ipynb

```diff
@@ -996,22 +996,215 @@
    "metadata": {},
```

Comment:
<p>It would be nice to also plot unfiltered vorticity.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NoraLoose/gcm-filters/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>